FRONTEND_CHANGE_TODO.md

```diff
@@ -0,0 +1,8 @@
+
+1. keep frontend patches - dont generate backend patches
```

Comment:
I think we still need to generate backend patches in the case where frontend and backend are on separate threads